apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -218,9 +218,9 @@ private async Task<int> ExecuteDdlAsync(CancellationToken cancellationToken)
                 {
                     var databaseAdminClient = new DatabaseAdminClientBuilder
                     {
-                        CallInvoker = channel.CreateCallInvoker(),
+                        // Note: deliberately not copying EmulatorDetection, as that's handled in SpannerClientCreationOptions
```

Comment:
Just maybe instead of just not copying, force it to not look for the emulator at all. Can't remember what was the default, and maybe the default already does that, in which case, it's fine.

Reply:
Yup, that's what the default does. We wanted users to *have* to be explicit :)